etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -131,4 +140,23 @@ DomainResponse handleOrders(DomainRequest request) {
         OrderResponse orderResponse = new OrderResponse(orders);
         return domainResponseHelper.constructOkResponse(orderResponse);
     }
+
+    DomainResponse handleMetadata(DomainRequest request) {
+        try {
+            String metadataId = request.getPathParams().get("id");
```

Comment:
Do we know if this is the submission id at this point? Or is this just an ID that will be determined later?

Reply:
It's the id RS is working on sending to us, which I think it'd be the original submissionId